content/en/monitors/create/types/apm.md

```diff
@@ -3,18 +3,18 @@ title: APM Monitor
 kind: documentation
 description: "Compare an APM metric to a user defined threshold"
 aliases:
-  - /monitors/monitor_types/app_analytics
-  - /monitors/monitor_types/trace_search
+  - /monitors/create/types/app_analytics
+  - /monitors/create/types/trace_search
   - /tracing/guide/resource_monitor/
-  - /monitors/monitor_types/apm
+  - /monitors/create/types/apm
```

Comment:
I think for aliases you want to keep the old ones (and maybe add the new ones? but definitely keep the old ones)